src/events/crawler/scrapers/USA/VA/index.js

```diff
@@ -0,0 +1,191 @@
+    'Greene County',
+    'Greensville County',
+    'Halifax County',
+    'Hampton city',
+    'Hanover County',
+    'Harrisonburg city',
+    'Henrico County',
+    'Henry County',
+    'Highland County',
+    'Hopewell city',
+    'Isle of Wight County',
+    'James City County',
+    'King and Queen County',
+    'King George County',
+    'King William County',
+    'Lancaster County',
+    'Lee County',
+    'Lexington city',
+    'Loudoun County',
+    'Louisa County',
+    'Lunenburg County',
+    'Madison County',
+    'Manassas city',
+    'Manassas Park city',
+    'Martinsville city',
+    'Mathews County',
+    'Mecklenburg County',
+    'Middlesex County',
+    'Montgomery County',
+    'Nelson County',
+    'New Kent County',
+    'Newport News city',
+    'Norfolk County',
+    'Northampton County',
+    'Northumberland County',
+    'Norton city',
+    'Nottoway County',
+    'Orange County',
+    'Page County',
+    'Patrick County',
+    'Petersburg city',
+    'Pittsylvania County',
+    'Poquoson city',
+    'Portsmouth city',
+    'Powhatan County',
+    'Prince Edward County',
+    'Prince George County',
+    'Prince William County',
+    'Pulaski County',
+    'Radford city',
+    'Rappahannock County',
+    'Richmond city',
+    'Richmond County',
+    'Roanoke city',
+    'Roanoke County',
+    'Rockbridge County',
+    'Rockingham County',
+    'Russell County',
+    'Salem city',
+    'Scott County',
+    'Shenandoah County',
+    'Smyth County',
+    'Southampton County',
+    'Spotsylvania County',
+    'Stafford County',
+    'Staunton city',
+    'Suffolk city',
+    'Surry County',
+    'Sussex County',
+    'Tazewell County',
+    'Virginia Beach city',
+    'Warren County',
+    'Washington County',
+    'Waynesboro city',
+    'Westmoreland County',
+    'Williamsburg city',
+    'Winchester city',
+    'Wise County',
+    'Wythe County',
+    'York County'
+  ],
+
+  async scraper() {
+    const pdfBaseURL = `${this.url}.pdf?:showVizHome=no&Locality=`;
+    const fullNameCounties = [
+      'Buena Vista city',
+      'Fairfax city',
+      'Franklin city',
+      'Franklin County',
+      'Manassas city',
+      'Richmond city',
+      'Richmond County',
+      'Roanoke city',
+      'Roanoke County'
+    ];
+    const counties = [];
+
+    for (const name of this._counties) {
+      const endURL = fullNameCounties.includes(name) ? name : name.slice(0, name.lastIndexOf(' '));
+      const pdfUrl = pdfBaseURL + endURL;
+      const pdfScrape = await fetch.pdf(pdfUrl);
+
+      let pdfText = '';
+      for (const item of pdfScrape) {
+        if (item.text === '©') {
+          break;
+        }
+        pdfText += item.text;
+      }
+
+      counties.push({
+        county: name,
+        cases: parse.number(pdfText.match(/(\d*)Cases/)[1]),
+        deaths: parse.number(pdfText.match(/(\d*)Deaths/)[1])
+      });
+    }
+
+    counties.push(transform.sumData(counties));
+
+    return counties;
```

Comment:
Same thing as above -- is this an exhaustive list of counties in VA?

Reply:
I'll add `geography.addEmptyRegions()` just in case